ci-operator/jobs/openshift/cluster-api-provider-openstack/openshift-cluster-api-provider-openstack-master-presubmits.yaml

```diff
@@ -82,6 +82,71 @@ presubmits:
         secret:
           secretName: registry-pull-credentials
     trigger: (?m)^/test( | .* )e2e-openstack,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - master
+    cluster: build01
+    context: ci/prow/e2e-upgrade
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-cluster-api-provider-openstack-master-e2e-upgrade
+    optional: true
+    path_alias: sigs.k8s.io/cluster-api-provider-openstack
+    rerun_command: /test e2e-upgrade
```

Comment:
I understand that this command exists in the context of CAPO. However in perspective, it would be great to run an upgrade test in the broader context of Installer. What about calling it `e2e-openstack-upgrade` so that we will trigger them with the same name in both repos?

Reply:
There is no need to add `-openstack-` in this test - there's no point to test CAPO on other platform. I could rename it for convenience.

Not sure if upgrade tests make sense in installer PRs - the workflow would use `stable-initial:installer` image to setup initial cluster, so changes in installer PR would not be used.
Verifying that installed cluster can be updated should be checked by `periodic-..-e2e-openstack-upgrade-4.7.ci` jobs

Reply:
OK, then the rename makes little sense. Thank you